.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+build/
```

Comment:
Generally speaking, anything generated should not be committed in git since it's easy for it to become out of date (e.g. if you forget to run `node build` before you commit). The workflow in this file will auto build all commits and deploy a pages app so we no longer need to commit the html files